nodejs_conductor/test/test-config.js

```diff
@@ -66,6 +63,84 @@ test('config construction', t => {
     t.equal(config.instances[1].agent, `bartolini`)
     t.equal(config.instances[1].dna, path)
     t.equal(config.interfaces.length, 0)
+}
+
+/**
+ * Brittle check for whether debug logging has been activated.
+ * Check this if a test is failing!
+ */
+const isFullDebugLogger = logger => logger.rules.rules.length > 1
+
+test('config construction, two argument, backwards compatible version', t => {
+    const path = 'path/to/dna'
+    const agent1 = C.agent('alessia')
+    const agent2 = C.agent('bartolini')
+    const dna = C.dna(path)
+    const config = C.conductor([
+        C.instance(agent1, dna),
+        C.instance(agent2, dna),
+    ])
+    console.log(config.logger.rules.rules)
+    commonConfigAssertions(t, config, path)
+    t.equal(config.bridges.length, 0)
+    t.notOk(isFullDebugLogger(config.logger))
+    t.end()
+})
+
+test('config construction, two argument, backwards compatible version with options', t => {
+    const path = 'path/to/dna'
+    const agent1 = C.agent('alessia')
+    const agent2 = C.agent('bartolini')
+    const dna = C.dna(path)
+    const config = C.conductor([
+        C.instance(agent1, dna),
+        C.instance(agent2, dna),
+    ], {
+        debugLog: true
+    })
+    commonConfigAssertions(t, config, path)
+    t.equal(config.bridges.length, 0)
+    t.ok(isFullDebugLogger(config.logger))
+    t.end()
+})
+
+test('config construction, single argument version', t => {
+    const path = 'path/to/dna'
+    const agent1 = C.agent('alessia')
+    const agent2 = C.agent('bartolini')
+    const dna = C.dna(path)
+    const config = C.conductor({
+        instances: [
+            C.instance(agent1, dna),
+            C.instance(agent2, dna),
+        ]
+    })
+    commonConfigAssertions(t, config, path)
     t.equal(config.bridges.length, 0)
+    t.notOk(isFullDebugLogger(config.logger))
+    t.end()
+})
+
+test('config construction, single argument version, with bridges and logger', t => {
+    const path = 'path/to/dna'
+    const agent1 = C.agent('alessia')
+    const agent2 = C.agent('bartolini')
+    const dna = C.dna(path)
+    const instance1 = C.instance(agent1, dna)
+    const instance2 = C.instance(agent2, dna)
+    const config = C.conductor({
+        instances: [instance1, instance2],
+        bridges: [
+            C.bridge('bridge-forward', instance1, instance2),
+            C.bridge('bridge-backward', instance2, instance1),
+        ],
+        debugLog: true,
+    })
+    commonConfigAssertions(t, config, path)
+    t.ok(isFullDebugLogger(config.logger))
+    t.deepEqual(config.bridges, [
+        {handle: 'bridge-forward', caller_id: 'alessia', callee_id: 'bartolini'},
```

Comment:
Just to make sure: the `caller_id` and `callee_id` here are instance_ids, not agent_ids, right?

Reply:
@juliolrmonteiro yes, this is the same as in the conductor config. It just so happens the instance IDs here are the same as the agent names :)